2022q2/portmgr.adoc

```diff
@@ -0,0 +1,38 @@
+=== Ports Collection
+
+Links: +
+link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[https://www.FreeBSD.org/ports/] +
+link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[Contributing to Ports] URL: link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[https://docs.freebsd.org/en/articles/contributing/#ports-contributing] +
+link:http://portsmon.freebsd.org/[FreeBSD Ports Monitoring] URL: link:http://portsmon.freebsd.org/[http://portsmon.freebsd.org/] +
+link:https://www.freebsd.org/portmgr/[Ports Management Team] URL: link:https://www.freebsd.org/portmgr/[https://www.freebsd.org/portmgr/] +
+link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[Ports Tarball] URL: link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[http://ftp.freebsd.org/pub/FreeBSD/ports/ports/]
+
+Contact: René Ladan <portmgr-secretary@FreeBSD.org> +
+Contact: FreeBSD Ports Management Team <portmgr@FreeBSD.org>
+
+//Show the importance of your work.
+//Status reports are not just about telling everyone that things were done, they also need to explain why they were done.
+
+The Ports Management Team is responsible for overseeing the overall direction of the Ports Tree, building packages, and personnel matters.
+Below is what happened in the last quarter.
+
+Freshports tells us the number of ports grew to over 50,000, a new milestone.
+The last quarter saw 9,137 commits by 151 committers on the "main" and 589 commits by 61 committers on the "2022Q2" branch.
+At the time of writing, there are 2,700 open ports PRs of which 682 are unassigned.
+Compared to the previous quarter, the Ports Tree grew by 7%, a slight decrease in commit activity and a constant number of PRs.
+
+During the last quarter, portmgr
+welcomed back salvadore@ but also said goodbye to seven ports committers due to lack of activity.
+
+In its bi-weekly meetings, portmgr discussed the following topics:
+* the future of ca_root_nss
+* feasibility of the base system providing certain .pc files
+* ways to deal with incompatibilities kernel module ports on minor version upgrades of the base system
+
+Following a discussion among developers, portmgr decided to grant all documentation and source
+committers approval to fix any documentation-related change in Ports Tree which does not affect its functionality.
```

Comment:
"in the Ports Tree"

Also, either "fix any documentation-related error" or "make any documentation-related change"

Reply:
Also fixed, thanks for the review.